spec/rubocop/cop/rails/lexically_scoped_action_filter_spec.rb

```diff
@@ -144,4 +144,18 @@ def foo; end
       end
     RUBY
   end
+
+  it "doesn't register an offense when using %I literal" do
+    expect_no_offenses <<-RUBY
+      class HogeController < ApplicationController
```

Comment:
"hoge" is dialect of Japan, so it would be better to prefer "foo", "bar", "baz" and so on.